CONTRIBUTING.md

```diff
@@ -0,0 +1,261 @@
+| Schema validation | In schema definition | Client-side validation |
+
+### Documentation Updates
+
+| When? | What to update | Command |
+|-------|----------------|---------|
+| New resource/data source | Add template + example | `make generatedocs` |
+| Change resource schema | Update template | `make generatedocs` |
+| Add/modify examples | Edit `.tf` files | `make generatedocs` |
+
+**Requirement**: Run `make generatedocs` and commit generated docs.
+
+Validate docs at: https://registry.terraform.io/tools/doc-preview
+
+### Changelog Entries
+
+| Change Type | When Required | Format |
+|------------|---------------|--------|
+| New feature | New resource/data source/field | `changelogs/fragments/YYYYMMDD-description.yml` |
+| Bug fix | Fixes existing behavior | `changelogs/fragments/YYYYMMDD-description.yml` |
+| Breaking change | Incompatible changes | Required + highlighted |
+| Internal refactoring | No user-facing change | Not required |
+
+**Tool:** This project uses [antsibull-changelog](https://github.com/ansible-community/antsibull-changelog) for changelog management.
+
+**Format:** Use existing fragments in `changelogs/fragments/` as examples.
+
+**Example fragment** (`changelogs/fragments/20251114-new-feature.yml`):
+```yaml
+minor_changes:
+  - "Added support for workflow job templates as a data source"
+```
+
+## CI Expectations
+
+| Check | Must Pass? | What it does |
+|-------|------------|--------------|
+| Unit tests | ✅ Yes | Runs `go test` |
+| Code coverage | ✅ Yes | Verifies ≥80% coverage on new code |
+| Linting | ✅ Yes | Runs `golangci-lint` |
+| Build | ✅ Yes | Compiles provider |
+
+**Policy**: PRs with failing CI will not be reviewed until fixed.
+
+## Review Process
+
+| Stage | Timeline | Action |
+|-------|----------|--------|
+| PR opened | Within 2 business days | Initial review |
+| Changes requested | N/A | Address feedback |
+| Approved | Within 1 business day | Merge to main |
+
+### What Reviewers Check
+
+- [ ] Tests pass and cover new code
+- [ ] Code coverage ≥80% for new code
+- [ ] Linting passes
+- [ ] Documentation is clear and complete
+- [ ] Code follows existing patterns
+- [ ] PR scope is focused
+- [ ] Changelog entry (if needed)
+
+## AI Code Assistant Usage
+
+If you use AI code assistants, please follow these guidelines:
+
+| Guideline | Details |
+|-----------|---------|
+| **Human verification** | Always review, test, and understand AI-generated code before submitting |
+| **When to attribute** | Note substantial AI assistance in PR description using `Assisted-by:` |
+| **Security** | Never input sensitive data (API keys, credentials, customer data) into AI tools |
+| **Code quality** | AI-generated code must meet the same standards (tests, coverage, linting) |
+| **License compliance** | Ensure AI suggestions don't introduce incompatible licenses |
+
+**Example PR description:**
+```
+Implemented retry logic for API calls
+
+Assisted-by: GitHub Copilot
```

Comment:
Good notes. We should also include any assitance attribution in the commit messages.